src/snapshots/001.txt

```diff
@@ -11,15 +11,13 @@ foo =
     "one"
 ~~~PROBLEMS
 NIL
-~~~TOKENS
-KwModule,OpenSquare,Newline,LowerIdent,Comma,Newline,CloseSquare,Newline,LowerIdent,OpAssign,Newline,StringStart,StringPart,StringEnd,EndOfFile
```

Comment:
I feel like we should always leave in raw tokens. Only make verbosity optional. But I would ask @joshuawarner32 what he prefers.